client/src/theme.js

```diff
@@ -100,6 +106,22 @@ const theme = createTheme({
             },
           },
         },
+        {
+          props: { variant: "rounded", color: "thai" },
+          style: {
+            textTransform: "none",
+            fontSize: "1rem",
+            borderRadius: "20px",
+            backgroundColor: "#98513A",
+            margin: ".4rem 0rem",
+            color: "#FFFFFF",
+            "&:hover": {
+              backgroundColor: "#98513A",
```

Comment:
Same question about hard coded hex colors. can we reference a constant for theme/brand colors if thats the case here? If we ever need to update colors in the future, we'll be searching for every reference, vs using a constant or a map to establish colors here, that would save some time and needed changes in future iterations

Reply:
i've asked PM to create a ticket to reconfigure theme. I don't think they have done it yet...should I just do it directly in this ticket then?

Reply:
I think we should continue to `DRY` up our code when opportunities exist. If it gets to the point where you think we need to refactor something. You should be able to create that ticket and write out your justification as well. PMs don't need to be the ones to write all the tickets, as a dev, you'll be writing plenty of tickets in the future too. Plus, doing all this work, you'll have the most context to provide more detail in said tickets

Reply:
Oh i thought devs didn't have access to create tickets (since we couldn't access the project board) that's why i asked PMs...just checked and it seems we can indeed create issues. that is my bad

Reply:
No worries, I think we may have changed the project from private to public at one point, I believe any github user can create tickets on the repo, but it would be up to us to kinda manage and make sure they're valid and ready to be worked on

Reply:
i'll reconfigure the theme now